h2o/h2o.sh

```diff
@@ -54,6 +54,11 @@ EOF
 }
 
 function main() {
+  if [[ "${DATAPROC_VERSION}" == "1.5" ]]; then
```

Comment:
Usually we put such conditions at very top of the init action: https://github.com/GoogleCloudDataproc/initialization-actions/blob/85e6216b9c0d9c6ea2f14b728b47862ab0e1bec1/conda/install-conda-env.sh#L4

Reply:
Fixed